### PR TITLE
Proofread Guidelines with minor edits

### PR DESCRIPTION
**Pull request by njwest**
### Description

Proofread and edited `guidelines/README.md`. Preserved meaning but clarified language a bit and fixed minor typos :)

🔥🚀🔥

### CODE REVIEW

File: guidelines/README.md [hunk 1]
```diff
@@ -32,76 +32,74 @@ Some other multimedia projects compilations:
...
 
-Arwes is an intent to materialize futuristic science fiction concepts in a toolset
-to let you create interfaces that inspire advanced technology, especially spatial,
+Arwes intends to materialize futuristic science fiction concepts in a toolset
+you may use to create interfaces inspired by advanced technology, especially spatial,
 influenced by alien conceptions.
 
-Connected, augmented, and energized are the feelings sought in the experiences
-provided with projects built with Arwes. So the interfaces are intensive and rich.
+Connected, augmented, and energized are the feelings sought in the experiences provided by projects built with Arwes, so Arwes interfaces are intensive and rich.
```
Comment (romelperez) [inline guidelines/README.md]: Can you set the character line limit to 80 please?

**Comment by romelperez** [comment]
Thank you for the corrections :) Just one comment and it looks great to merge.

**Comment by romelperez** [comment]
Also, can you change the git commit messages format to `docs: message`?

File: guidelines/README.md (hunk 1 above)
Comment (njwest) [reply guidelines/README.md]: 👍

**Comment by njwest** [comment]
Will do!

**Pull request merged**